generators/module/templates/package

```diff
@@ -13,7 +13,8 @@
     "eslint-fix": "npm run eslint -- --fix",
     "prepublish": "rollup -c",
     "test": "run-s testonly eslint",
-    "testonly": "jest"
+    <% if (codecov) { %>"test-travis": "jest --coverage && codecov",
```

Comment:
put this at the end of the preceding line to avoid having a blank line when the option is false